src/main/scala/mesosphere/marathon/core/pod/PodInstanceStateOp.scala

```diff
@@ -0,0 +1,13 @@
+package mesosphere.marathon.core.pod
+
+import mesosphere.marathon.core.instance.{ Instance, InstanceStateOp }
+
+sealed trait PodInstanceStateOp extends InstanceStateOp
+
+object PodInstanceStateOp {
+
+  case class LaunchedEphemeral(podInstance: PodInstance) extends PodInstanceStateOp {
```

Comment:
@meichstedt do we need to differentiate between pods and apps for Launch(ed)Ephemeral, or should I nix this type in favor of TaskStateOp.LaunchEphemeral?